JavaScript-Runtime-(JSRT)-Reference.md

```diff
@@ -138,3 +139,20 @@ References of JSRT typedefs, constants, enumerations and APIs.
 * [[JsSetRuntimeMemoryLimit]]
 * [[JsStrictEquals]]
 * [[JsStringToPointer]]
+
+### New string API (Only non-Windows)
```

Comment:
'Cause this https://github.com/Microsoft/ChakraCore/blob/master/lib/Jsrt/ChakraCore.h#L166. 
I'm not sure,